### DISCUSSION TITLE
SMTP Set sender name

### DISCUSSION THREAD

**Discussion by sdobbelaere** [chatwoot/chatwoot#4340] [Feature Requests]
All emails sent out by chatwoot are sent out as the email used in the smtp configuration.  
It would be much cleaner in terms of branding if the sender name would also be configurable next to the email-address.

This would make the emails received by our clients much more professional.

**Comment by sojan-official** [comment]
@sdobbelaere  thanks for the feedback. lets track this over https://github.com/chatwoot/chatwoot/issues/4315
